src/Entity/MessageTemplate.php

```diff
@@ -253,9 +262,23 @@ public function getText($langcode = Language::LANGCODE_NOT_SPECIFIED, $delta = N
       $text = $translated_text ?: [];
     }
 
+    // Process text format.
+    foreach ($text as $key => $item) {
+      // Call the renderer directly instead of adding a dependency on the Filter
```

Comment:
Yeah, although I've added the `text` module as a dependency, which in turn depends on `filter`, but I'm inclined to leave this as-is for easier unit testing (otherwise, for any unit test, we'd need to redeclare that function in the test).